minio/sidekick
# support cert/key reload without server restart

support cert/key reload without server restart
fix https://github.com/minio/sidekick/issues/106
how to test:
```shell
PS C:\workspace\go\src\sidekick\dist> ./certgen -host "192.168.2.6,127.0.0.1,localhost"             
Created a new certificate 'public.crt', 'private.key' valid for the following names 📜
 - "192.168.2.6"
 - "127.0.0.1"
 - "localhost"
PS C:\workspace\go\src\sidekick\dist> mc alias set sk https://192.168.2.6:8090 minioadmin minioadmin
Fingerprint of sk public key: 2385a69ce94fb9f70e9a53fd5a91310c41c93a394e9ff659877310e20ce7d57b
Confirm public key y/N: y
mc.exe: <ERROR> Unable to initialize new alias from the provided credentials. Get "https://192.168.2.6:8090": tls: failed to verify certificate: x509: certificate signed by unknown authority.
PS C:\workspace\go\src\sidekick\dist> ./certgen -host "127.0.0.1,localhost"                         
Created a new certificate 'public.crt', 'private.key' valid for the following names 📜
 - "127.0.0.1"
 - "localhost"
PS C:\workspace\go\src\sidekick\dist> mc alias set sk https://192.168.2.6:8090 minioadmin minioadmin
mc.exe: <ERROR> Unable to initialize new alias from the provided credentials. Get "https://192.168.2.6:8090": tls: failed to verify certificate: x509: certificate is valid for 127.0.0.1, not 192.168.2.6.
PS C:\workspace\go\src\sidekick\dist> 
```
